zhaquirks/thirdreality/plug.py

```diff
@@ -0,0 +1,123 @@
+"""Third Reality Plug devices."""
+
+from typing import Final
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.smartenergy import Metering
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+from zhaquirks import CustomCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.thirdreality import THIRD_REALITY
+
+THIRD_REALITY_PLUG_CLUSTER_ID = 0xFF03
+RESET_SUMMATION_DELIVERED_ATTR_ID = 0x0000
+
+
+class ControlMode(t.uint8_t):

```

Comment:
The `NOT` and `CLEAR` parts don't really have an effect, as this is not an enum.
There's a longer discussion here: https://github.com/zigpy/zha-device-handlers/pull/2723#discussion_r1487114673

I'd just remove this section for now and set `type=t.uint8_t` for the attribute.
(When the button is later exposed to HA, it'll be clear that `1` means reset.)

Reply:
OK